### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Phantomkeks**
### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: www/docs/de/6.x/guide/overview/index.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ toc_title: Overview
 
 # Übersicht
 
-Apache Cordova ist ein Open-Source-mobile-Entwicklung-Framework. Sie können standard-Web-Technologien wie HTML5, CSS3 und JavaScript für Cross-Plattform-Entwicklung, Vermeidung jeder mobilen Plattformen native Entwicklung der Sprache zu verwenden. Anwendungen werden in Verpackungen, die gezielt auf jede Plattform und verlassen sich auf standardkonforme API Anbindungen an jedes Gerät Sensoren, Daten und Netzwerkstatus zugreifen.
+Apache Cordova ist ein Open-Source-Framework für die mobile Software-Entwicklung. Es ermöglicht, unter der Verwendung von Standard-Web-Technologien wie HTML5, CSS3 und JavaScript, die Entwicklung von Cross-Plattform-Anwendungen, ohne dabei native Entwicklungssprachen  der mobilen Plattformen nutzen zu müssen. Anwendungen werden in sogenannte Wrapper verschalt, die gezielt auf die jeweilige Plattform angepasst sind und somit den Zugriff auf Sensoren, Daten und den Netzwerkstatus eines Gerätes durch standardkonforme Schnittstellen erlauben.
```
Comment (piotrowski) [inline www/docs/de/6.x/guide/overview/index.md]: verschalt?
Vorschlag: verpackt

File: www/docs/de/6.x/guide/overview/index.md [hunk 2]
```diff
@@ -23,19 +23,19 @@ toc_title: Overview
...
 
-Apache Cordova graduierte Oktober 2012 als Top-Level-Projekt innerhalb der Apache Software Foundation (ASF). Durch die ASF wird künftige Cordova offene Leitung des Projekts sichergestellt. Es bleibt immer kostenlos und open Source unter der Apache License, Version 2.0. Besuchen Sie [cordova.apache.org][1] für weitere Informationen.
+Apache Cordova erreichte im Oktober 2012 den Top-Level-Projektstatus innerhalb der Apache Software Foundation (ASF). Durch die ASF wird die künftige Entwicklung offene Leitung des Projekts sichergestellt. Dabei bleibt Cordova kostenlos und untersteht als Opensource Porjekt der Apache License, Version 2.0. Besuchen Sie [cordova.apache.org][1] für weitere Informationen.
```
Comment (piotrowski) [inline www/docs/de/6.x/guide/overview/index.md]: offene => und
Opensource => Open Source
Porjekt => Projekt
License => Lizenz

File: www/docs/de/6.x/guide/overview/index.md [hunk 3]
```diff
@@ -23,19 +23,19 @@ toc_title: Overview
...
 
  [1]: http://cordova.apache.org
 
-Verwenden Sie Apache Cordova, falls Sie sind:
+Verwenden Sie Apache Cordova, wenn Sie:
 
-*   mobile Entwickler und wollen eine Anwendung über mehrere Plattformen hinweg zu erweitern, ohne es erneut mit Sprache und Tool jede Plattform implementieren festgelegt.
+*   ein mobiler Anwendungsentwickler sind und eine Anwendung über mehrere Plattformen hinweg verbreiten möchten, ohne für jede Plattform  eine separate Anwendungsversion in nativer Sprache implementieren müssen.
```
Comment (piotrowski) [inline www/docs/de/6.x/guide/overview/index.md]: müssen => zu müssen

**Comment by shazron** [comment]
@piotrowski I don't speak German. Is this good to go?

**Comment by filmaj** [comment]
Wouldn't this need to be merged into the 7.x content as well, to preserve moving forward?

**Comment by piotrowski** [comment]
@shazron Yes, great improvement on the current text.

**Comment by shazron** [comment]
Should be both in the 7.x and dev trees as well.

**Comment by janpio** [comment]
As this is sitting here: Done correctly this should probably be done in Crowdin directly, not in the source. I am not sure this would be synced back to Crowdin this way. But, currently it is also unclear if the changes done in Crowdin would really get deployed, so best wait until this situation is fixed.

**Pull request closed without merging**